src/components/TopBar/TopBarMenu.vue

```diff
@@ -79,10 +79,13 @@
 		</NcActionButton>
 
 		<!-- Fullscreen -->
-		<NcActionButton :icon="iconFullscreen"
```

Comment:
You can delete the iconFullscreen computed too?

Reply:
Indeed. My omission - a bit embarrassing. Thanks for pointing this out. :+1: Done in https://github.com/nextcloud/spreed/pull/9484/commits/c9adfe233edd86b4892580b981ecf0fc1eb9b15b

I am a bit concerned about the `isInCall` check that was in the removed code. It seems that the icon is always in a menu with a light background - even during a call. So, I hope this will be fine.
![obraz](https://user-images.githubusercontent.com/8277636/236640779-43ce49fd-2d3a-4dac-8b9d-da09a1da7c3b.png)

Reply:
> It seems that the icon is always in a menu with a light background - even during a call

When using dark mode, the menu is dark as well:
![image](https://user-images.githubusercontent.com/1580193/236672378-c2edb8b2-4b74-4ff8-bd46-db1ec65d7e1b.png)

Reply:
I have checked in dark mode. It is OK.

In conversation:
![obraz](https://user-images.githubusercontent.com/8277636/236697368-dca6f6c7-6181-41f5-866e-05522d5b233a.png)

During a call:
![obraz](https://user-images.githubusercontent.com/8277636/236697408-1494dd7f-776f-41d7-874f-d4f97c6865c1.png)